README.md

```diff
@@ -28,8 +28,12 @@ This project adheres to a [code of conduct](CODE_OF_CONDUCT.md). By participatin
 
 Many thanks to the wonderful contributors who've helped create and improve this series.
```

Comment:
Maybe something like:

Special thanks to Zannah Marsh ([@zee-moz](https://github.com/zee-moz)) and Chad Sansing ([@chadsansing](https://github.com/chadsansing)) who've played a big role in creating this series. Many thanks to the wonderful contributors who've improved this series: